docs/sdk/v3/guides/liquidity/03-collecting-fees.md

```diff
@@ -0,0 +1,70 @@
+---
+id: liquidity-fees
+title: Collecting Fees
+---
+
+## Introduction
+
+This guide will cover how to collect fees from a liquidity position on the Uniswap V3 protocol.
+It is based on the [collecting fees code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/collecting-fees), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/collecting-fees/README.md) and follow the setup instructions.
+
+:::info
+
+If you need a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background](./01-background.md) page!
+
+:::
+
+In the Uniswap V3 protocol, liquidity positions are represented using non-fungible tokens. In this guide we will use the `NonfungiblePositionManager` class to help us mint a liquidity position for the  **USDC - DAI** pair. We will then attempt to collect any fees that the position has accrued from those trading against our provisioned liquidity. The inputs to our guide are the **two tokens** that we are pooling for, the **amount** of each token we are pooling for, the Pool **fee** and the **amount of accrued fees** we want to collect for each token.
+
+The guide will **cover**:
+1. Creating or minting our position using the `NonfungiblePositionManager`'s `addCallParameters` to get the data for creting the position minting transaction, and then executing the transaction.
+2. Collecting accrued fees from our position by using the `NonfungiblePositionManager`'s `collectCallParameters` to get the data for creating the collect fees transaction, and then executing the transaction.
+
+At the end of the guide, given the inputs above, we should be able to mint a liquidity position with the press of a button and view the position's id on the UI of the web application. We should also be able to collect the accrued fees (if any) with the press of a button and see the change reflected in the balance of our tokens.
+
+## Example
+
+### Creating or minting our position by using the `NonfungiblePositionManager`'s `addCallParameters` to get the data for making the position minting transaction, and then executing the transaction
```

Comment:
Since this is already covered in the previous guide which we link to, why are we covering this? We should link only and focus the example on getting the fees alone